api/v1alpha1/groupversion_info.go

```diff
@@ -14,29 +14,23 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// NOTE: Boilerplate only.  Ignore this file.
-
-// Package v1alpha1 contains API Schema definitions for the openebs v1alpha1 API group
-// +k8s:deepcopy-gen=package,register
-// +groupName=openebs.io
+// Package v1alpha1 contains API Schema definitions for the  v1alpha1 API group
+//+kubebuilder:object:generate=true
+//+groupName=openebs.io
 package v1alpha1
 
 import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/controller-runtime/pkg/scheme"
 )
 
-const (
-	// GroupName is the group of apis
-	GroupName = "openebs.io"
-	// APIVersion is the version for the apis
-	APIVersion = "v1alpha1"
-)
-
 var (
-	// SchemeGroupVersion is group version used to register these objects
-	SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: APIVersion}
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "openebs.io", Version: "v1alpha1"}
```

Comment:
Was the change from constants to using string literals directly done by the auto generated code?

Reply:
It was boilter plate code IIRC